README.md

```diff
@@ -1,2 +1,55 @@
 # err-topic-a-day
-An errbot plugin to post a topic a day to a channel for discussion
+
+## Description
+This Bot (http://errbot.io) posts to slack  plugin to post a topic a day to a channel for discussion
+
+## Usage
+TODO: FILL THIS
+
+## Validation
+When running the program, some validations will run and if they are not configured properly an exception will be raised with the appropriate respone
+
+# Slack commands
+These are the command you can send privately to the bot and he will respond apprpriately
+
+## add_topic
+send it the topic and the topic name and it will add it to the back of the queque
+
+## list_topics
+shows all the topics in all of the states:
+- used (previous)
+- unused (upcoming)
+
+# Variables
+I guess the usage is by environment variables and `.env` files, 
+TODO: verify this with owner
+
+## Customizable
```

Comment:
Add some notes that these are configured as environment variables?

Reply:
I tried to answer your question by adding an explanation on how to configure environment variables.. please tell me what needs to change.

Reply:
These config options have to be set as environment variables. How you do that changes a bit based on how you are deploying your errbot. I think it would be fine to say something along the lines of "These options are configured via environment variables" and leave it at that.

Reply:
I hope I fixed it now =]